src/components/CorporateCampaign/LoanSearch/LoanSearchFilters.vue

```diff
@@ -488,6 +534,24 @@ export default {
 			if (this.selectedSort !== sortBy) {
 				this.selectedSort = sortBy;
 			}
+		},
+		handleResetFilters() {
+			this.modifiedFilters = this.copyFilters(this.initialFilters);
```

Comment:
Why was this needed, were the filters set hanging around with just the emit? The emitted event should handle resetting things...

Reply:
Ok. I see this is exactly the case so good deal